src/general/Mutations.cs

```diff
@@ -81,7 +82,7 @@ public MicrobeSpecies CreateMutatedSpecies(MicrobeSpecies parent, MicrobeSpecies
             mutated.IsBacteria = false;
         }
 
-        var colour = mutated.IsBacteria ? RandomProkayroteColour() : RandomColour();
+        var colour = mutated.IsBacteria ? RandomProkayroteColour() : RandomCellColour();
```

Comment:
I think `RandomCellColour` should be named `RandomEukaryoteColour` to make it clear that it is a counterpart to `RandomProkayroteColour`

Also I just noticed a typo in `Prokayrote` could you fix that in this file as well?